bookstore/bookstore.py

```diff
@@ -1,29 +1,94 @@
+#return dictionary with {'name': name}
 def create_bookstore(name):
-    pass
-
+    rmotr_bookstore = {}
```

Comment:
Yeah I'm going to take that approach moving forward. Definitely more tidy and easy on the eyes. Also probably helps to visualize it when initializing for subsequent steps. Thanks Jason!